example/ci/pipeline.yaml

```diff
@@ -110,4 +109,4 @@ jobs:
         params:
           build: source/docker
           dockerfile: source/docker/Dockerfile
-          additional_tags: s3-gitversion/version
+          tag: s3-gitversion/version
```

Comment:
This is wrong. The `tag` section expects a string. So your image would be tagged with `s3-gitversion/version`. Also the usage of `tag` is deprecated. Use `tag_file` instead.

See https://github.com/concourse/docker-image-resource for reference.

Reply:
did work in my test, though?! 
but I will adapt to `tag_file`.